README.md

```diff
@@ -10,104 +10,150 @@
   
-2. Go to **System Console > Plugins > GitHub** and do the following:
-  - Generate a new value for **At Rest Encryption Key**.
-  - (Optional) **GitHub Organization:** Lock the plugin to a single GitHub organization by setting this field to the name of your GitHub organization.
-  - (Optional) **Enable Private Repositories:** Allow the plugin to receive notifications from private repositories by setting this value to true.
-    When enabled, existing users must reconnect their accounts to gain access to private repositories. Affected users will be notified by the plugin once private repositories are enabled.
+Open **System Console > Plugins > GitHub** and do the following:
+
+1. Generate a new value for **At Rest Encryption Key**.
+2. (Optional) **GitHub Organization:** Lock the plugin to a single GitHub organization by setting this field to the name of your GitHub organization.
+3. (Optional) **Enable Private Repositories:** Allow the plugin to receive notifications from private repositories by setting this value to `true`.
+4. (**Enterprise Only**) **Enterprise Base URL** and **Enterprise Upload URL**: Set these values to your GitHub Enterprise URLs, e.g. `https://github.example.com`. The Base and Upload URLs are often the same. When enabled, existing users must reconnect their accounts to gain access to private repositories. Affected users will be notified by the plugin once private repositories are enabled.
+5. Hit **Save**.
+6. Go to **System Console > Plugins > Management** and click **Enable** to enable the GitHub plugin.
+
+You're all set!
+
+## Using the Plugin
+
+Once configuration is complete, run the `/github connect` slash command from any channel within Mattermost to connect your Mattermost account with GitHub. The command and its output isn't visible to other members of the channel.
+
+## Onboarding Your Users
 
-  - (Enterprise only) **Enterprise Base URL** and **Enterprise Upload URL**: Set these values to your GitHub Enterprise URLs, e.g. `https://github.example.com`. The Base and Upload URLs are often the same.
-3. Hit **Save**.
-4. Go to **System Console > Plugins > Management** and click **Enable** to enable the GitHub plugin.
+When you’ve tested the plugin and confirmed it’s working, notify your team so they can connect their GitHub account to Mattermost and get started. Copy and paste the text below, edit it to suit your requirements, and send it out.
 
-You're all set! To test it, run the `/github connect` slash command to connect your Mattermost account with GitHub.
+> Hi team, 
+
+> We've set up the Mattermost GitHub plugin, so you can get notifications from GitHub in Mattermost. To get started, run the `/github connect` slash command from any channel within Mattermost to connect your Mattermost account with GitHub. The command is only visible to you.. Then, take a look at the [slash commands](https://github.com/mattermost/mattermost-plugin-github#slash-commands) section for details about how to use the plugin.
```

Comment:
What does `The command is only visible to you` mean?

Reply:
When you type the command in a channel, other members of the channel don't see it afaik. I was a bit worried about that when I first set it up. We do also reference that in other documentation (which I'll need to find to cross-check).

Reply:
Should we talk about the command-response then? The term `command` usually refers to the slash command i.e. the user input.

Reply:
Ah yes, that works much better.